Assets/Scripts/Managers/SaveManager.cs

```diff
@@ -0,0 +1,166 @@
+﻿using System;
+using System.Collections.Generic;
+using Buildings;
+using Entity.Player;
+using GameTime;
+using Interactables;
+using Inventory;
+using UnityEngine;
+using UnityEngine.AI;
+using Utils.Saves;
+
+namespace Managers
+{
+    public class SaveManager : Singleton<SaveManager>
+    {
+        /// <summary>
+        /// saves the current state of the game to JSON
+        /// </summary>
+        /// <param name="fileName"></param>
+        public void Save(string fileName)
+        {
+            try
+            {
+                // grab required components to extract data from
+                GameObject player = PlayerManager.Instance.GetPlayer();
+                InventoryController inventoryController = InventoryManager.Instance.GetInvController();
+
+                // this is the player's current position in the map
+                Vector3 playerPosition = player.transform.localPosition;
```

Comment:
Why don't you use transform.position? Is there a difference in this case?

Reply:
I also think it should be `position` instead of `localPosition`

Reply:
the position that changes when the player moves is the localPosition. See the debug view in the Unity editor.